packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -324,7 +324,7 @@ class TextMapPropagator {
           spanContext = this._extractB3MultiContext(carrier)
           break
         default:
```

Comment:
Since you're already in a switch statement based on the extractor value you could make this a lot cleaner by doing this:
```js
        case 'baggage':
          break
        default:
          log.warn(`Unknown propagation style: ${extractor}`)
```

Reply:
It's kind of awkward having baggage as a propagator, if we go with this cleaner solution (which also makes more sense to me), in the case we have `this._config.tracePropagationStyle.extract = ['baggage', 'datadog']`, we would end up with `spanContext` without `traceId` or `parentId` which the `datadog` propagation style clearly requires. Having baggage as a propagator simply means we are propagating baggage, but it should not override other propagation styles. We've decided on having baggage as a propagator since otel does so.